### ISSUE TITLE
Unique Port Name Runtime exception

### ISSUE THREAD

**Issue by jachin** [joeandaverde/elm-webpack-project-loader#13]
tldr; We brought 2 different elm Mains together and they both had a port called `init` and everything complied _fine_, `elm-webpack-project-loader` brought them together and then it blew up at run time.

We have just started switching over to using `elm-webpack-project-loader` and so far it's looking great.

We did run into 1 issue though, and it's totally "work around-able" and probably not even this project's fault, but it still may make sense to try to do something about it at this layer.

It would appear this is not a new issue: https://github.com/elm/compiler/issues/1450

In theory it seems like this is something `elm make` should catch.

However, maybe it's possible to do something in this loader to check for conflicting port names and stop it at the webpack level?

**Comment by joeandaverde** [comment]
Thanks for the analysis. Any ideas on what can be done besides using unique port names?
